dali/test/python/test_operator_nonsilence.py

```diff
@@ -80,7 +80,7 @@ def test_nonsilence_operator():
     batch_size = 3
     window_sizes = [512, 1024]
     reset_intervals = [-1, 2048, 8192]
-    references_power = [None, .0003]
+    references_power = [None, .03]
```

Comment:
The previous threshold was too small (the whole recording was non-silent, always). This is a better test